pkg/clustermanager/manager.go

```diff
@@ -194,7 +194,7 @@ func (m *Manager) toRESTConfig(cluster *v3.Cluster) (*rest.Config, error) {
 
 	rc := &rest.Config{
 		Host:        u.Host,
-		Prefix:      u.Path,
+		APIPath:     u.Path,
```

Comment:
I don't think Prefix and APIPath are the same.  I believe it should be `Host: u.String()` and APIPath is left empty